cmd/swarmctl/node/common.go

```diff
@@ -138,6 +138,16 @@ func getNode(ctx context.Context, c api.ControlClient, input string) (*api.Node,
 		}
 
 		if l := len(rl.Nodes); l > 1 {
+			for _, n := range rl.Nodes {
+				name := n.Spec.Annotations.Name
```

Comment:
@stevvooe The fact is that there is no way for user to set a name for a node explicitly. Is there any historical reason on this? And should we file a separate issue for this?